### DISCUSSION TITLE
J4.2.2 Joomla displays 301 Status code for 404 Pages

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#47110] [2. Cold Features]
### Steps to reproduce the issue
Turn on URL Collecting in the redirect Component and go to your Joomla Website and type in something not existing after the domain, something like yourdomain.com/sudlsdblsdvblhsldh

### Expected result
The site is recorded as 404 Error

### Actual result
Site is recorded with 301 Status Code 

<img width="1504" alt="grafik" src="https://user-images.githubusercontent.com/828371/191086522-9095b35c-6420-4750-8a64-08c1eac5ce6d.png">

### System information (as much as possible)
I tested this on J.4.2.2 Core, no extensions, just Cassiopeia
Also tested on other websites with different templates with same result

**Comment by drmenzelit** [comment]
In Joomla 3.10.11 is the same
![grafik](https://user-images.githubusercontent.com/9153168/191094635-64f7a639-1c75-428d-8a92-8f8c16b0b3fc.png)

**Comment by coolcat-creations** [comment]
Appears that the table shows "the future" of the entry and not the real status code. I would recommend to write 404 in the status unless a redirect is added, then it can show 301.
